default.json

```diff
@@ -366,6 +366,16 @@
         "patch",
         "digest"
       ]
+    },
+    {
+      "matchPackageNames": ["rancher/kuberlr-kubectl"],
+      "matchUpdateTypes": ["major"],
+      "enabled": false
```

Comment:
Just realized this might be redundant and that just using the second rule might be enough?

Reply:
I think you are right, I proposed the change below. Doing a small test now to confirm it.

Reply:
Yep, that seems to be able to do the trick.